docs/sources/administration/data-source-management/team_lbac.md

```diff
@@ -0,0 +1,158 @@
+
+Configuring multiple rules for a team, each rule is evaluated separately. If a team has `X` number of rules configured for it, all rules will be applied to the request and the result will be the an "OR" operation of the `X` number of rules.
+
+Only users with data source Admin permissions can edit LBAC rules at the data source permissions tab. Changing LBAC rules requires the same access level as editing data source permissions (admin permission for data source).
+
+> "Can I use CAPs (cloud access policies) together with TeamLBAC rules?"
+> No, CAP (cloud access policies) always have precedence. If there are any CAP LBAC configured for the same datasource and there are TeamLBAC rules configured, then only the CAP LBAC will be applied.
+
+Cloud access policies are the access controls from Grafana Cloud, the CAP configured for loki should only to be used to gain read access to the logs.
+
+> "If administrator forget to add rule for a team, what happens?"
+> The teams that does not have a rule applied to it, would be able to query all logs if `query` permissions are setup for their role within Grafana.
+
+**Note:** A user who is part of a team within Grafana without a rule will be able to query all logs if there are role based queriying setup.
+
+#### Best practices
+
+We recommend you only add team LBAC permissions for teams that should use the data source and remove default `Viewer` and `Editor` query permissions.
+
+We recommend for a first setup, setting up as few rules for each team as possible and make them additive and not negated.
+
+For validating the rules, we recommend testing the rules in the Loki Explore view. This will allow you to see the logs that would be returned for the rule.
+
+#### Scenarios
+
+**Scenario 1: One rule setup for each team**
+
+We have two teams, Team A and Team B. Loki access is setup with `Admin` roles to have `Admin` permission only.
+
+- Team A has a rule `namespace="dev"`.
+
+- Team B has a rule `namespace="prod"`.
+
+A user that is part of Team A will have access to logs that match `namespace="dev"`.
+
+A user that is part of Team B will have access to logs that match `namespace="prod"`.
+
+**Scenario 2: Multiple rules setup for one team**
+
+We have two teams, Team A and Team B. Loki access is setup with `Admin` roles having `Admin` permission.
+
+- Team A has rule `cluster="us-west-0", namespace="dev|prod"` configured.
+
+- Team B has rule `cluster="us-west-0", namespace="!prod"` configured.
+
+A user that is only part of Team A will have access to logs that match `cluster="us-west-0" AND (namespace="dev" OR namespace="prod")`.
+
+A user that is only part of Team B will have access to logs that match `cluster="us-west-0" AND namespace!="prod"`.
+
+A user that is part of both Team A and Team B will have access to logs that match `cluster="us-west-0" AND (namespace="dev" OR namespace="prod") OR (is this true?) (cluster="us-west-0" AND namespace!="prod")`.
```

Comment:
It's might be easier to understand if we write it in a simplier form. Team A has access to logs in cluster `us-west-0` with namespaces `dev` and `prod`. Team B has access to to everything in cluster `us-west-0`, except namespace `prod`. So basically, user who is member of both team A and team B has access to everything in cluster `us-west-0`.